src/styles/messages.scss

```diff
@@ -391,7 +292,7 @@
 	&__title {
 		display: inline-block;
 		@include unselectable;
-		font-size: 13px;
+		font-size: 11px;
```

Comment:
The same thing as with color value. It's better to extract this to a dedicated property

Reply:
same commit: https://github.com/th2-net/th2-rpt-viewer/commit/4f08c9d2f2a05b20a8ecfcced428a53c826c6992